opentelekomcloud/acceptance/ecs/data_source_opentelekomcloud_compute_instance_v2_test.go

```diff
@@ -37,7 +37,6 @@ func TestAccComputeV2InstanceDataSource_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "name", "instance_1"),
 					resource.TestCheckResourceAttrPair(resourceName, "metadata", instanceName, "metadata"),
 					resource.TestCheckResourceAttrSet(resourceName, "network.0.name"),
-					resource.TestCheckResourceAttrSet(resourceName, "encrypted_password"),
```

Comment:
is it required?

Reply:
encrypted_password is not required attribute

Reply:
here we don't assert only `required` attrs